src/content/app/entity-viewer/variant-view/population-allele-frequencies/PopulationAlleleFrequencies.tsx

```diff
@@ -14,18 +14,25 @@
  * limitations under the License.
  */
 
-import { useState, useEffect } from 'react';
-import classNames from 'classnames';
-
 import { createSmallNumberFormatter } from 'src/shared/helpers/formatters/numberFormatter';
 
-import usePopulationAlleleFrequenciesData from './usePopulationAlleleFrequenciesData';
+import usePopulationAlleleFrequenciesData, {
+  PreparedPopulationFrequencyData
+} from './usePopulationAlleleFrequenciesData';
 
 import Panel from 'src/shared/components/panel/Panel';
 import { CircularProportionIndicator } from 'src/shared/components/proportion-indicator/CircularProportionIndicator';
 import { Table, ColumnHead } from 'src/shared/components/table';
 import { CircleLoader } from 'src/shared/components/loader';
 
+import {
+  Accordion,
+  AccordionItem,
+  AccordionItemHeading,
+  AccordionItemButton,
+  AccordionItemPanel
+} from 'src/shared/components/accordion';
```

Comment:
In general, the accordion is a complex component, with an api that might feel cumbersom; I try to avoid it wherever I can. [Here](http://allele-frequency-ui.review.ensembl.org/entity-viewer/grch38/gene:ENSG00000221914?view=transcripts), for instance, we have a `ShowHide` component in the expanded transcript panel (see also the [ShowHide component in Storybook](https://ensembl.github.io/ensembl-client/storybook/index.html?path=/story/components-shared-components-showhide--show-hide-story)). Wouldn't it make more sense to use that?